tests/src/resultcollection.hh

```diff
@@ -0,0 +1,170 @@
+// SPDX-FileCopyrightText: 2022 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+#include "testcommon.hh"
+
+#include <ikarus/finiteelements/ferequirements.hh>
+
+template <typename FiniteElement>
+auto getVertexPositions(FiniteElement& fe) {
+  constexpr int dim            = FiniteElement::Traits::mydim;
+  const auto& element          = fe.gridElement();
+  const auto& referenceElement = Dune::referenceElement<double, dim>(element.type());
+  const int nodes              = referenceElement.size(dim);
+
+  std::vector<typename FiniteElement::GridElementEntityType::Geometry::LocalCoordinate> positions;
+  for (auto i : std::views::iota(0, nodes))
```

Comment:
You can also write here:

```cpp
std::ranges::transform(std::views::iota(0, nodes), positions, std::back_inserter(positions), [&](int i){return referenceElement.position(i, dim);})
```

Since I have the impression you like ranges and i.e. `std::views::iota` ;)

Reply:
Haha i think in this case, the non range variant is a bit more clear :D